pkg/vm/vm.go

```diff
@@ -217,7 +219,14 @@ func (v *VM) PrintOps(out io.Writer) {
 				if utf8.Valid(parameter) {
 					desc = fmt.Sprintf("%x (%q)", parameter, parameter)
 				} else {
-					desc = fmt.Sprintf("%x", parameter)
+					// Try converting the parameter to an address and swap the endianness
+					// if the parameter is a 20-byte value.
+					u, err := util.Uint160DecodeBytesBE(parameter)
+					if err == nil {
+						desc = fmt.Sprintf("%x (Address:%q, Swap Endianness:%q)", parameter, address.Uint160ToString(u), "0x"+hex.EncodeToString(slice.CopyReverse(parameter)))
```

Comment:
I'd avoid "address:" and "swap endianness:", the first one is obvious, the second is also not that hard to understand (Neo uses "0x..." notation for stringified reversed hashes all over the place).

Reply:
Fixed.